Library/Homebrew/formula.rb

```diff
@@ -923,9 +957,23 @@ def exec_cmd(cmd, args, out, logfn)
     exit! 1 # never gets here unless exec threw or failed
   end
 
+  def source_dir
+    if build.dsym?
+      prefix + "Source"
+    else
+      nil
+    end
+  end
```

Comment:
Ok. The option (and PR description) should be updated to reflect that all the source code is also copied.

Perhaps a better approach here is to initialise a Git repository in the source directory and copy the source files using `git archive master | tar -x -C /somewhere/else` or similar. That would mean you aren't reliant on out-of-source builds.